docs/guides/tools.md

```diff
@@ -42,3 +42,66 @@ type Query {
   myName: String!
 }
 ```
+
+---
+
+### `depth_limit_validator`
+
+Create a validator to limit the complexity of queries by their depth to protect against malicious
+queries.
+
+```python
+from graphql import ValidationRule
```

Comment:
can we make our own class for this? 🤔 

to make sure we are abstracted from GraphQL core

Reply:
I'm not sure there would be much benefit to that because it would have to be a subclass of the `ValidationRule` class anyway for this check to work: https://github.com/graphql-python/graphql-core/blob/03c2b06e4636ed8a89c7a4c7e56d244fd9d00bde/src/graphql/validation/validate.py#L51-L56

Reply:
uhm ok :) we could convert it as we do we the schema converter, but that's fine for now :)